Refactor code

Assult72 opened the pull request:
The branch has done:
- Changed name of some method, variable and property to an appropriate name.
- Removed unnecessary code.
- Write window prompt information directly in method body, instead of assigning it to the constant.
- Changed the logic of getting FeatureService url and other service url.
- Changed updating mechanism of ElevationSourceAddedChanged and ElevationSourceRemovedChanged

yong7743 commented on ToArcGISEarth/ToArcGISEarth/ConnectButton.cs:
```diff
@@ -17,73 +17,56 @@ namespace ToArcGISEarth
                 {
-                    this.Enabled = true;
-                    if (this.IsChecked)
+                    string result = await ToolHelper.Utils.Connect();
```
The method should return a bool instead of a string.

yong7743 commented on ToArcGISEarth/ToArcGISEarth/RemoveLayerButton.cs:
```diff
@@ -47,44 +48,44 @@ protected override void OnUpdate()
         }
 
-        private void RemoveLayer(LayerEventsArgs args)
+        private void RemoveLayerFromEarthEvent(LayerEventsArgs args)
```
This is a method not event, so please don't use Event as a name for the method.

Assult72 commented:
- Updated some method name
- Fixed the issue that removing a source, and the same name source from another surface will be removed in ElevationSufaces.
![snipaste_2018-09-27_19-34-55](https://user-images.githubusercontent.com/24773248/46143573-fff11600-c28c-11e8-8eff-0fdad893a058.png)
